changelog.md

```diff
@@ -7,6 +7,8 @@
 * Added parameter `allow_overwrite` to [`lk.download_from_doi()`](https://lumicks-pylake.readthedocs.io/en/latest/_api/lumicks.pylake.download_from_doi.html#lumicks.pylake.download_from_doi) to allow re-downloading only those files where the checksum does not match.
 * Added force calibration information to channels accessed directly via the square bracket notation (e.g. `file["Force HF"]["Force 1x"].calibration`).
 * Calibration results and parameters are now accessible via properties which are listed when items are printed. See [calibration results](https://lumicks-pylake.readthedocs.io/en/latest/_api/lumicks.pylake.force_calibration.power_spectrum_calibration.CalibrationResults.html) and [calibration item](https://lumicks-pylake.readthedocs.io/en/latest/_api/lumicks.pylake.calibration.ForceCalibrationItem.html) API documentation for more information.
+* Added `applied_at` property to [calibration item](https://lumicks-pylake.readthedocs.io/en/latest/_api/lumicks.pylake.calibration.ForceCalibrationItem.html) which returns the timestamp in nanoseconds at which the force calibration was applied.
```

Comment:
I don't think two user facing specializations is that bad, since they serve different purposes and one cannot be substituted for the other. They are always going to have a few properties the other one doesn't have.

For calibration items, they're gonna be timestamps. For calibration results they're going to be the ability to graph and validate the results.

I do think the naming right now is kind of terrible though (see previous comment). But yeah, let's discuss tomorrow. I can quickly set up a branch in the morning where some things are shuffled around to hopefully make things a bit more apparent. I didn't want to move stuff yet in the middle of relatively big changes (since it complicated review), but maybe I should have.